rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -710,33 +726,47 @@ public void run() {
               delayNanos,
               TimeUnit.NANOSECONDS,
               scheduledExecutorService);
+      logger.log(ChannelLogLevel.DEBUG,
+          "BackoffCacheEntry created with a delay of {0}s",
+          TimeUnit.NANOSECONDS.toSeconds(delayNanos));
```

Comment:
We may want to wrap this in an if to avoid the computation when disabled.

Reply:
Unfortunately we don't expose the set log level via `ChannelLogger`. I switch to simply using nanos without the conversion to seconds. Overkill to have nanos in the log but I can't think of a better compromise.

Reply:
That should be fine.